src/stdatm/speed_parameters.py

```diff
@@ -10,3 +11,249 @@
 #  GNU General Public License for more details.
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from functools import singledispatch
+from numbers import Real
+
+import numpy as np
+from scipy.optimize import root
+
+from stdatm.state_parameters import GAMMA
+
+
+# TRUE AIRSPEED =================================================================
+# From Mach -----------------------------------------------------------------
+def compute_tas_from_mach(mach, speed_of_sound):
+    """
+
+    :param mach: no unit
+    :param speed_of_sound: in m/s
+    :return: true airspeed in m/s
+    """
+    true_airspeed = mach * speed_of_sound
+    return true_airspeed
+
+
+# From EAS -----------------------------------------------------------------
+def compute_tas_from_eas(equivalent_airspeed, density, sea_level_density):
+    """
+
+    :param equivalent_airspeed: in m/s
+    :param density: in kg/m**3
+    :param sea_level_density: in kg/m**3
+    :return: true airspeed in m/s
+    """
+    true_airspeed = equivalent_airspeed * np.sqrt(sea_level_density / density)
+    return true_airspeed
+
+
+# From unitary Reynolds -----------------------------------------------------------------
+def compute_tas_from_unit_re(unitary_reynolds, kinematic_viscosity):
+    """
+
+    :param unitary_reynolds: in 1/m
+    :param kinematic_viscosity: in m**2/s
+    :return: true airspeed in m/s
+    """
+    true_airspeed = unitary_reynolds * kinematic_viscosity
+    return true_airspeed
+
+
+# From dynamic pressure -----------------------------------------------------------------
+def compute_tas_from_pdyn(dynamic_pressure, density):
+    """
+
+    :param dynamic_pressure: in Pa
+    :param density: in kg/m**3
+    :return: true airspeed in m/s
+    """
+    true_airspeed = np.sqrt(2.0 * dynamic_pressure / density)
+    return true_airspeed
+
+
+# MACH =================================================================
+def compute_mach(true_airspeed, speed_of_sound):
+    """
+
+    :param true_airspeed: in m/s
+    :param speed_of_sound: in m/s
+    :return: Mach number (no unit)
+    """
+    mach = true_airspeed / speed_of_sound
+    return mach
+
+
+# EQUIVALENT AIRSPEED =================================================================
+def compute_equivalent_airspeed(true_airspeed, density, sea_level_density):
+    """
+
+    :param true_airspeed: in m/s
+    :param density: in kg/m**3
+    :param sea_level_density: in kg/m**3
+    :return: equivalent airspeed in m/s
+    """
+    equivalent_airspeed = true_airspeed / np.sqrt(sea_level_density / density)
```

Comment:
Something I did not catch last time, and I know I'm gonna sound even pickier than for the gamma but why not 
```suggestion
    equivalent_airspeed = true_airspeed * np.sqrt(density / sea_level_density )
```